mlserver/batch_processing.py

```diff
@@ -88,8 +88,12 @@ def json_to_triton(
     inference_request = InferenceRequest.parse_obj(orjson.loads(request_json))
     for request_input in inference_request.inputs or []:
         new_input = httpclient.InferInput(
-            request_input.name, request_input.shape, request_input.datatype
+            request_input.name,
+            request_input.shape,
+            request_input.datatype,
         )
+        if request_input.parameters is not None:
+            new_input._parameters.update(request_input.parameters.dict())
```

Comment:
Good catch! 

We initially considered leveraging the `_parameters` field, but then decided not to, in order to avoid using Tritonclient's internal interface. Instead of that, the general advise when using `mlserver infer` is to just configure the `content_type` through the model's metadata (i.e. the `model-settings.json` file). 

Would be great to hear your thoughts behind this change though.